website/components/navigation/Navigation.tsx

```diff
@@ -158,16 +145,23 @@ function Navigation() {
   }
 
   return (
-    <nav className={navContainer}>
+    <SideNav
```

Comment:
One thing I'm noticing is that when you hover over the collapsed nav it pushes the layout over slightly, even though it's being overlayed on top of the page content. It's particularly noticeable on the home page

Reply:
There also might be a better way to transition between the tablet/mobile nav and the desktop nav

Reply:
Turned out to be an issue with the scroll bar. Fixed!